src/angular/11-declarative-state/declarative-state.md

```diff
@@ -46,81 +45,75 @@ everything about how the state is created. In imperative code you'd need to read
 
 Code using declarative state is often shorter than imperative code since you're not needing to write as much flow control logic.
 
-Declarative state can be less error / side effect prone since it's typically more specific about how state is generated
-relative to imperative code which may modify state under conditions which may at first seem correct, but end up having
-unintended consequences.
+Declarative state can be less error prone. It's typically more specific about how state is generated relative to 
+imperative code, which may modify state under conditions which may at first seem correct, but end up having unintended
+consequences. 
 
-An additional benefit of Angular + RxJS is that imperative side effects, like those that we previously accomplished in a
-subscription to an event (_e.g_ toggling a form control to be enabled / disabled) can now be done in an action taken
-during the generation of a value by a stream. Avoiding subscriptions removes the need to manage them in `onDestroy`.
+An additional benefit of Angular + RxJS is that declarative state can be used directly in the template, removing the need
+for most subscriptions in our component. Avoiding subscriptions eliminates the need to manage them in `onDestroy`.
 
-## Essential RxJS operators
-RxJS operators are the actions that occur to generate values in a stream. There are dozens of operators that do things
+## Using BehaviorSubject To Observe Form Control Values
+<!-- TODO: behavior subject --> 
+
+## Essential RxJS Operators & Functions
+RxJS operators are the actions that occur to generate values in a stream. There are dozens of operators that do things 
 like transform values, filter values, combine streams, and much more. We'll just be touching on a small selection
 of operators.
 
+We'll also demonstrate several important static functions that RxJS provides to create and combine streams. 
+
+### Creating A Stream
+<!-- TODO: of --> 
+
+### Combining Streams
+<!-- TODO: merge / combineLatest -->
+
 ### Initializing A Stream
 A common situation is working with streams that only produce a value after an event, for example when an HTTP request
 completes or when a value changes in a form control. When using a stream like this in your components, you'll likely
-want to have an initial "base state" that your view can use during the initial render. In RxJS this is handled by the
+want to have an initial "base state" that your view can use during the initial render. In RxJS this is handled by the 
 `startWith` operator, which emits a value when the stream is first subscribed to.
```

Comment:
To stay in line with Angular docs, we usually include links to official docs of stuff like this. = )